_episodes/02-metadata.md

```diff
@@ -0,0 +1,102 @@
+to review your submitted publication, to replicate your results, to design a
+similar study, or even to archive your data for access and
+re-use by others. While digital data by definition are machine-readable,
+understanding their meaning is a job for human beings. The
+importance of documenting your data during the collection and analysis phase of
+your research cannot be overestimated, especially if your
+research is going to be part of the scholarly record.  
+
+However, metadata should not be contained in the data file itself. Unlike a table
+in a paper or a supplemental file, metadata (in the
+form of legends) should not be included in a data file since this information is
+not data, and including it can disrupt how computer
+programs interpret your data file. Rather, metadata should be stored as a
+separate file in the same directory as your data file,
+preferably in plain text format with a name that clearly associates it with your
+data file. Because metadata files are free text format,
+they also allow you to encode comments, units, information about how null values
+are encoded, etc. that are important to document but can
+disrupt the formatting of your data file.  
+
+Some of this information may be familiar to learners who conduct analyses on
+survey data or other data sets that come with codebooks. Codebooks will often
+describe the way a variable has been constructed, what prompt was associated with
+it in an survey or interview, and what the meaning of various values are. For example,
+the [General Social Survey](http://gss.norc.org) maintains their entire codebook online.
+Looking at an entry for a particular variable, such as
+[the variable `SEX`](https://gssdataexplorer.norc.org/variables/81/vshow), provides
+valuable information about what survey waves the variable covers, and the meaning
+of particular values.
+
+Additionally, file or database level metadata describes how files that make up
+the dataset relate to each other; what format are they are
+in; and whether they supersede or are superseded by previous files. A
+folder-level readme.txt file is the classic way of accounting for
+all the files and folders in a project.  
+
+Metadata are most useful when they follow a standard. For example, the
+[Data Documentation Initiative (DDI)](http://www.ddialliance.org) provides a
+standardized way to document metadata at various points in the research cycle.
+Research librarians may have specific expertise in this area, and can be
+helpful resources for thinking about ways to purposefully document metatdata
+as part of your research.
+
+(Text on metadata adapted from the online course Research Data [MANTRA](http://datalib.edina.ac.uk/mantra) by EDINA and Data Library, University of Edinburgh. MANTRA is licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/).)
```

Comment:
Most other carpentries lessons don't have referencing in this form? is there a specific style we should use to be more consistent (ie on a separate page or is it because more of the content is generic?

Reply:
that is a great question. I don't believe there is... @ErinBecker - can you confirm that references like this are not standardized across DataCarpentry lessons?

Reply:
There's no guidance in the style guide, I think it would be a good thing to add to the  [style guide](http://carpentries.github.io/lesson-example/06-style-guide/index.html) of [lesson example](https://github.com/carpentries/lesson-example/blob/gh-pages/_episodes/06-style-guide.md)